src/main.rs

```diff
@@ -19,23 +23,62 @@ struct Flock {
     time_per_frame: i32,
 }
 
+fn validate_distances(max_dist_before_boid_is_crowded: f32, max_dist_of_local_boid: f32) -> Result<(), CreationError> {
```

Comment:
ifl `max_dist_before_boid_is_no_longer_crowded` is what I was trying to say, which makes more sense to me, when considering x,y increasing away from the boid rather than towards, will rename to that